code/style.css

```diff
@@ -47,6 +47,20 @@ input {
   flex: 1;
 }
 
+select {
+  box-sizing: border-box;
+  border: none;
+  border-radius: 4px 0 0 4px;
+  background-color: #0026ff;
+  color: white; 
+  padding: 16px;
+  font-size: 16px;
+  font-family: 'Montserrat';
+  font-weight: 600;
+  line-height: 26px;
+  text-align: center; 
+}
```

Comment:
Nice to see that you styled your generated selectMenu, as a further development in coming projects, perhaps you could try adding it via Javascript as well 🥰?